tools/lint-all

```diff
@@ -83,10 +83,10 @@ def check_pep8(files):
     failed = False
     ignored_rules = [
         'E402', 'E501', 'W503', 'E711', 'E201', 'E203', 'E202', 'E128', 'E226', 'E124', 'E125',
-        'E126', 'E127', 'E121', 'E122', 'E123', 'E266', 'E265', 'E261', 'E301', 'E221', 'E303',
-        'E241', 'E712', 'E225', 'E401', 'E115', 'E114', 'E111', 'E222', 'E731', 'E302', 'E129',
-        'E741', 'E714', 'W391', 'E211', 'E713', 'E502', 'E131', 'E305', 'E251', 'E306', 'E231',
-        'E701', 'E702',
+        'E126', 'E127', 'E121', 'E122', 'E123', 'E266', 'E265', 'E261', 'E301', 'E221', 'E241',
+        'E712', 'E225', 'E401', 'E115', 'E114', 'E111', 'E222', 'E731', 'E302', 'E129', 'E741',
+        'E714', 'W391', 'E211', 'E713', 'E502', 'E131', 'E305', 'E251', 'E306', 'E231', 'E701',
+        'E702',
```

Comment:
No, just remove the item, rather than also adjusting the following lines (which I assume you did in order to keep the number of items on each line constant, but doing so isn't important).